lib/iris/coord_systems.py

```diff
@@ -820,3 +820,51 @@ def as_cartopy_crs(self):
 
     def as_cartopy_projection(self):
         return self.as_cartopy_crs()
+
+
+class Mercator(CoordSystem):
+    """
+    A coordinate system in the Mercator projection.
+
+    """
+
+    grid_mapping_name = "mercator"
+
+    def __init__(self, central_lon=0, ellipsoid=None):
+        """
+        Constructs a Mercator coord system.
+
+        Args:
+            * central_lon
+                    The central longitude.
+
+        Kwargs:
+
+            * ellipsoid
+                    :class:`GeogCS` defining the ellipsoid.
+
+        """
+
+        #: True longitude of planar origin in degrees.
```

Comment:
If you don't want them documented (probably no need, but I don't really mind) then remove the `:`.